src/safeds/ml/classical/regression/_lasso_regression.py

```diff
@@ -13,9 +14,25 @@
 
 
 class LassoRegression(Regressor):
-    """Lasso regression."""
-
-    def __init__(self) -> None:
+    """Lasso regression.
+
+    Parameters
+    ----------
+    alpha : float, default=1.0
+        Constant that multiplies the L1 term. Defaults to 1.0.
+
+    Raises
+    ------
+    ValueError
+        If alpha is negative.
+    """
+
+    def __init__(self, alpha: float = 1.0) -> None:
+        if alpha < 0:
+            raise ValueError("alpha must be non-negative")
+        if alpha == 0:
+            warn("alpha is zero, you should use LinearRegression instead", UserWarning, stacklevel=1)
```

Comment:
```suggestion
            warn("Setting alpha to zero makes this model equivalent to LinearRegression. You should use LinearRegression instead for better numerical stability.", UserWarning, stacklevel=1)
```

I think this better explains why this is an issue.